### ISSUE TITLE
Button warning and button danger styles don't meet color accessibility requirements

### ISSUE THREAD

**Issue by immihaylova** [vmware/clarity#2701]
## Describe the bug

The usage of #E62700 foreground color with #FAFAFA background color fails on color contrast check.

## How to reproduce

Run a contrast checker on the warning and danger outline butons. Easiest way is to use the buttons from the docs: https://vmware.github.io/clarity/documentation/v0.13/buttons.

1.  Go to '...'
2.  Click on '....'
3.  Scroll down to '....'
4.  See error

## Expected behavior

The contrast check should pass.

## Versions

**App**

* Clarity: [0.11, 0.12, 0.13]

**Device:**

* Type: [workstation]
* OS: [Win10x63]
* Browser [chrome]
* Version [e.g. 22]

## Additional notes

_Add any other notes about the problem here._

**Comment by mathisscott** [comment]
Hi @immihaylova 
I think this is similar to the issue we encountered in [this accessibility issue with the blue color](https://github.com/vmware/clarity/issues/2585). The challenge is our contrast ratio was tested against #FFFFFF but not against the off-white background on the content area.

**Comment by mathisscott** [comment]
@immihaylova 
I will see what I can do about 0.11
